corehq/apps/app_manager/templates/app_manager/partials/case_list_lookup.html

```diff
@@ -0,0 +1,91 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<div id="{{ detail.type }}-list-callout-configuration" data-bind="with: caseListLookup">
+    <legend>
+        {% trans "Case List Search Callout" %}
+    </legend>
+    <label class="checkbox">
+        <input type="checkbox" data-bind="checked: lookup_enabled"/>
+        {% trans "Enable case list search callout" %}
+    </label>
+    <div data-bind="visible: lookup_enabled">
+        <form class="form-horizontal">
+            <div class="control-group">
+                <label class="control-label" for="lookup-action">
+                    {% trans "Action" %}
+                </label>
+                <div class="controls">
+                    <input type="text" id="lookup-action" data-bind="value: lookup_action" placeholder="{% trans 'ex: callout.commcarehq.org.dummycallout.LAUNCH' %}" required/>
```

Comment:
Since this partial may be included on the page multiple times (once for each "detail". Multiple details exist in `AdvancedModule`s), you should avoid using raw html ids. I think the pattern on this page is to do something like this instead:
```html
<input id="{{ detail.type }}-lookup-action" />
```
The annoying part about this is that you then need to pass the `detail.type` to the corresponding view model to be used in the jQuery selectors.